tools/apprunner-gradle-plugin/build.gradle

```diff
@@ -4,7 +4,12 @@ plugins {
   id 'com.gradle.plugin-publish' version '0.12.0'
 }
 
-version = file('version.txt').text.trim()
+def quarkusVersion = file('../../pom.xml').readLines().find {l -> l.contains('<quarkus.version>') }.replaceFirst(~/.*<quarkus.version>(.*)<\/quarkus.version>.*/, '$1' )
```

Comment:
How does this work for plugin users?

Reply:
Users don't use this build.gradle (it's just for building and publishing the plugin), they just "apply" the published plugin.